.github/workflows/terraform.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Terraform'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+    
+env:
+  TF_VAR_api_url: "https://api.london.cloud.service.gov.uk"
+  TF_VAR_user: ${{ secrets.TF_VAR_user }}
+  TF_VAR_password: ${{ secrets.TF_VAR_password }}
+  TF_VAR_SECRET_KEY_BASE: ${{ secrets.TF_VAR_SECRET_KEY_BASE }}
+  TF_VAR_SENTRY_DSN: ${{ secrets.TF_VAR_SENTRY_DSN }}
+  TF_VAR_SETTINGS__GOOGLE__GCP_API_KEY: ${{ secrets.TF_VAR_SETTINGS__GOOGLE__GCP_API_KEY }}
+  TF_VAR_SETTINGS__GOOGLE__MAPS_API_KEY: ${{ secrets.TF_VAR_SETTINGS__GOOGLE__MAPS_API_KEY }}
+  BACKEND_STORAGE_ACCOUNT_NAME: "s121d01tfstatestr"
+  BACKEND_ACCESS_KEY: ${{ secrets.BACKEND_ACCESS_KEY }}
+  BACKEND_KEY: "paas-find.tfstate"
+
+jobs:
+  terraform:
+    name: 'Terraform deployment for PASS'
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    # Checkout the repository to the GitHub Actions runner
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Run bash
+      run: |
+          echo $HOME
+          terraform version
+
+    - name: Install Terraform CloudFoundry Provider
+      run: |
+          wget https://github.com/cloudfoundry-community/terraform-provider-cf/releases/download/v0.12.2/terraform-provider-cloudfoundry_linux_amd64
```

Comment:
You can use option `--directory-prefix=$HOME/.terraform.d/plugins/linux_amd64` to download directly to the right place